From 7b5370b95bac41e2aecbaf4d7118e6f42cdb1714 Mon Sep 17 00:00:00 2001
From: Abhay Krishna Arunachalam <arnchlm@amazon.com>
Date: Tue, 10 Aug 2021 21:49:17 -0700
Subject: [PATCH] Replace promptui mod to mitigate LGPL dependency

Flux has a dependency on the manifoldco/promptui module, which in turn has a
dependency on juju/ansiterm module, which is LGPLv3 licensed. There is currently a 
PR to remove this dependency (https://github.com/manifoldco/promptui/pull/181) that
that has not been merged, but the PR author has updated their own fork to remove this
dependency, so we are using that module instead of the one from manifoldco.

Signed-off-by: Abhay Krishna Arunachalam <arnchlm@amazon.com>
---
 go.mod | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/go.mod b/go.mod
index 0d918ab..1ac4488 100644
--- a/go.mod
+++ b/go.mod
@@ -35,3 +35,5 @@ require (
 	sigs.k8s.io/kustomize/api v0.7.4
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/manifoldco/promptui => github.com/nguyer/promptui v0.8.1-0.20210517132806-70ccd4709797
-- 
2.30.1

